docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,482 @@
+harness-deployment  . -u -dtls -l -d azathoth.local -e local -n azathoth -i samples
+----
+
+Pay attention at what's displayed.
+At the end of the output, you'll encounter a line like this one:
+
+[source,bash]
+----
+To test locally, update your hosts file
+X.X.X.X	azathoth.local samples.azathoth.local hub.azathoth.local sentry.azathoth.local accounts.azathoth.local common.azathoth.local volumemanager.azathoth.local argo.azathoth.local workflows.azathoth.local notifications.azathoth.local events.azathoth.local
+----
+
+Where `X.X.X.X` will be a dedicated IP address.
+Insert this line into your `hosts` file, and your good to go for the build/deployment.
+
+[NOTE]
+If you missed this line, you can run the previous `harness-deployment` command a second time line, or you can find the IP address launching `minikube ip`.
+
+Once your configuration is created, you can build/deploy/run all the services/apps using `skaffold`.
+Skaffold will connect to the local docker environnement inside your minikube cluster to build all the images.
+Obviously, this step takes time.
+
+.Building/deploying/running all services/apps
+[source,bash]
+----
+skaffold run
+----
+
+Now that everything is deployed and running, you can see the sample page by going to `http://samples.azathoth.local`.
+Of course, this address depends on what you used as domain name, and entirely relies on the modification of your `hosts` file.
+
+
+You can monitor the state of all of your apps and services using `minikube`'s dashboard.
+
+.Checking the state of the cluster and running apps/services
+[source,bash]
+----
+minikube dashboard
+----
+
+This command will launch a page in your browser that provides all the information you need for your minikube cluster.
+
+
+== Creating a very simple webapp
+
+Now that we know how to configure/run/deploy apps on our local cluster, we will create a very simple webapp.
+In this first time, we will only generate the project's artifacts using the `harness-application`, then, we will build/run/deploy it.
+In a second time, we will modify the API to add new endpoints and deal with the frontend accordingly.
+
+=== Creating a new webapp
+
+The webapp that we will create will be a useless webapp that will fetch the current date and time when a button is pressed.
+Nothing fancy, just a way to see how to interact with the generated sources and get everything running on your local cluster.
+
+The first step is to generate the projects files.
+In our case, we want to develop a webapp, meaning that we want a frontend and a backend.
+We use `harness-application` to generate the first files with a specific tempates: `webapp` and `server`.
+We first place ourself in the parent directory of where you cloned the `cloud-harness` repository.
+
+[NOTE]
+We could place ourself anywere, we would just have to remember the path towards the `cloud-harness` repository.
+
+.Generating the first project's file
+[source,bash]
+----
+harness-application clockdate -t webapp -t server
```

Comment:
Thanks to your tutorial we were able to spot a [bug](https://github.com/MetaCell/cloud-harness/issues/558). Thanks :)
I'm mentioning this here not only to appreciate your work but also because in the bug comments you can check that '-t server' will most likely need to be renamed to '-t flask-server' (pending confirmation)

Reply:
I'll change the `server` template for `flask-server`.
We saw with @filippomc yesterday that the `webapp` template isn't generating the server part. It looks like a regression. I'll open an issue for that.

Reply:
Thanks for the review @afonsobspinto ! :D